jira_integration/jira_integration.php

```diff
@@ -0,0 +1,111 @@
+<?php
+// TODO: this header should not be required...
+header('Content-Type: text/plain; charset=UTF-8');
+
+// Only watch the dev environment
+if ($_ENV['PANTHEON_ENVIRONMENT'] != 'dev') {
```

Comment:
I'm thinking back to the conversation about these being `examples` vs engineering full blown solutions. I think as an example just watching dev would be sufficient and prevent over-complicating the spirit of an example.